<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GM-Assistant</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../ressource.qrc">
    <normaloff>:/data/images/GMA.svg</normaloff>:/data/images/GMA.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="GridLayout">
      <item row="4" column="0">
       <widget class="QGroupBox" name="gbScenario">
        <property name="title">
         <string>Scenario</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout1">
         <item>
          <widget class="QCustomTreeWidget" name="treeScenario">
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::InternalMove</enum>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::MoveAction</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="animated">
            <bool>true</bool>
           </property>
           <property name="headerHidden">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QGroupBox" name="gbHistory">
        <property name="title">
         <string>History</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout2">
         <item>
          <widget class="QCustomTreeWidget" name="treeHistory">
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::InternalMove</enum>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::MoveAction</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="animated">
            <bool>true</bool>
           </property>
           <property name="headerHidden">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QGroupBox" name="gbNote">
        <property name="title">
         <string>Notes</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout4">
         <item>
          <widget class="QCustomTextEdit" name="textNotes">
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="undoRedoEnabled">
            <bool>false</bool>
           </property>
           <property name="acceptRichText">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QGroupBox" name="gbCharacter">
        <property name="title">
         <string>Characters</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout5">
         <item>
          <widget class="QCustomTableWidget" name="tableStats">
           <property name="verticalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QGroupBox" name="gbSound">
        <property name="title">
         <string>Sound effects</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout6">
         <item>
          <widget class="QCustomTreeWidget" name="treeFX">
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::InternalMove</enum>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::MoveAction</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="animated">
            <bool>true</bool>
           </property>
           <property name="headerHidden">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QGroupBox" name="gbMusic">
        <property name="title">
         <string>Music</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout3">
         <item>
          <widget class="QCustomTreeWidget" name="treeMusic">
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::InternalMove</enum>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::MoveAction</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="animated">
            <bool>true</bool>
           </property>
           <property name="headerHidden">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QCustomSlider" name="sliderMusic">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="statusTip">
              <string>Browse the music file</string>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout1">
             <item>
              <widget class="QPushButton" name="buttonMusic">
               <property name="statusTip">
                <string>Play/Pause/Resume the music</string>
               </property>
               <property name="text">
                <string>&amp;Play</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelPosition">
               <property name="text">
                <string>0:00/0:00</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkRepeat">
               <property name="statusTip">
                <string>Enable/Disable loop playing</string>
               </property>
               <property name="text">
                <string>&amp;Loop</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>878</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>&amp;Game</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="separator"/>
    <addaction name="action_Reload"/>
    <addaction name="action_Load"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="actionS_ave_as"/>
    <addaction name="separator"/>
    <addaction name="actionR_ecent"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;?</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionFull"/>
    <addaction name="actionSimple"/>
    <addaction name="actionMusic"/>
    <addaction name="actionDesign"/>
    <addaction name="actionNoMusic"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="action_Dice"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="statusTip">
    <string>Create a new game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Load">
   <property name="text">
    <string>&amp;Load</string>
   </property>
   <property name="statusTip">
    <string>Load a game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="statusTip">
    <string>Save the current game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionS_ave_as">
   <property name="text">
    <string>S&amp;ave as</string>
   </property>
   <property name="statusTip">
    <string>Save the current game in a new file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="statusTip">
    <string>Exit the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="statusTip">
    <string>Display information about GM-Assistant</string>
   </property>
  </action>
  <action name="action_Reload">
   <property name="text">
    <string>&amp;Reload</string>
   </property>
   <property name="statusTip">
    <string>Reload the current game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionR_ecent">
   <property name="text">
    <string>R&amp;ecent</string>
   </property>
   <property name="statusTip">
    <string>Load a recently opened game</string>
   </property>
  </action>
  <action name="actionFull">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Full</string>
   </property>
   <property name="statusTip">
    <string>Tools shown : Scenario, Note, Characters, History, Music, Sound</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
   <property name="softKeyRole">
    <enum>QAction::NoSoftKey</enum>
   </property>
  </action>
  <action name="actionMusic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Music</string>
   </property>
   <property name="statusTip">
    <string>Tools shown : Music, Sound</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionSimple">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Simple</string>
   </property>
   <property name="statusTip">
    <string>Tools shown : Scenario, Music and Sound</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionDesign">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Design</string>
   </property>
   <property name="statusTip">
    <string>Tools shown : Scenario, Characters, Note </string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionNoMusic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;No Music</string>
   </property>
   <property name="statusTip">
    <string>Tools shown : Scenario, Note, History, Characters</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="statusTip">
    <string>Undo the last modification</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="statusTip">
    <string>Redo the last undone modification</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="action_Dice">
   <property name="text">
    <string>&amp;Dice simulator</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>QCustomTreeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>QCustomTableWidget</class>
   <extends>QTableWidget</extends>
   <header>QCustomTableWidget.h</header>
  </customwidget>
  <customwidget>
   <class>QCustomSlider</class>
   <extends>QSlider</extends>
   <header>QCustomSlider.h</header>
  </customwidget>
  <customwidget>
   <class>QCustomTextEdit</class>
   <extends>QTextEdit</extends>
   <header>QCustomTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../ressource.qrc"/>
 </resources>
 <connections/>
</ui>
